client/lxd_storage_volumes.go

```diff
@@ -34,7 +35,7 @@ func (r *ProtocolLXD) GetStoragePoolVolumeNames(pool string) ([]string, error) {
 }
 
 // GetStoragePoolVolumeNamesAllProjects returns the names of all volumes in a pool for all projects.
-func (r *ProtocolLXD) GetStoragePoolVolumeNamesAllProjects(pool string) ([]string, error) {
+func (r *ProtocolLXD) GetStoragePoolVolumeNamesAllProjects(pool string) (map[string][]string, error) {
```

Comment:
@stgraber this changes the public API signature of this function, but the original implementation of this function would never have worked and would always return an error, so are you happy to change this now?

Reply:
Yeah, that's fine, if no downstream could ever have used it successfully, then we're not likely to break anything.